README.md

```diff
@@ -6,11 +6,28 @@ Python support for Sublime's LSP plugin provided through [microsoft/pyright](htt
 
 1. Install [LSP](https://packagecontrol.io/packages/LSP) and
    [LSP-pyright](https://packagecontrol.io/packages/LSP-pyright) via Package Control.
-1. Restart Sublime.
+2. Restart Sublime.
+3. Configure pyright for your virtualenv.
```

Comment:
I don't think we have to manually assign the list number and it's a bad practices imo.